src/components/Reports/GDPRPermissions/PermissionExpandableRow.js

```diff
@@ -61,37 +99,89 @@ const PermissionExpandableRow = ({ field, dependencies: { experimentalFeatures }
       </tr>
 
       {isPermissionHistoryEnabled && (
-        <tr className={`dp-expanded-table ${expanded && 'show'} dp-table-responsive`}>
-          <>
-            <td className="dp-latest-results">
-              <FormattedMessage id="subscriber_gdpr.latest_results" tagName="span" />
-            </td>
-            <td className="dp-list-results">
-              <table className="dp-table-results">
-                <thead>
-                  <tr>
-                    <th aria-label={_('subscriber_gdpr.consent')} scope="col">
-                      <FormattedMessage id="subscriber_gdpr.consent" tagName="span" />:
-                    </th>
-                    <th aria-label={_('subscriber_gdpr.modification_source_ip')} scope="col">
-                      <FormattedMessage
-                        id="subscriber_gdpr.modification_source_ip"
-                        tagName="span"
-                      />
-                      :
-                    </th>
-                    <th aria-label={_('subscriber_gdpr.modification_date')} scope="col">
-                      <FormattedMessage id="subscriber_gdpr.modification_date" tagName="span" />:
-                    </th>
-                    <th aria-label={_('subscriber_gdpr.source_form')} scope="col">
-                      <FormattedMessage id="subscriber_gdpr.source_form" tagName="span" />:
-                    </th>
-                  </tr>
-                </thead>
-              </table>
+        <tr className={`dp-expanded-table ${expanded && 'show'}`}>
+          {loading ? (
+            <>
+              <td />
+              <td>
+                <S.EmptyBox>
+                  <Loading />
+                </S.EmptyBox>
+              </td>
+              <td />
+            </>
+          ) : unexpectedError ? (
+            <td className="dp-unexpected-error-table" colSpan={3}>
+              <span>
+                <span className="dp-icon-warning" /> ¡Ouch! Hubo un problema de conexión.
+                <a href="#" onClick={reloadRowData}>
```

Comment:
Si no es realmente un link, sino un botón, usamos un button estilado como link.

```suggestion
                <button onClick={reloadRowData}>
```

Reply:
Si, es una buena practica. Lo reemplazo.